### PR TITLE
Document 'copy_upstream_release_description' config option

### PR DESCRIPTION
**Pull request by lbarcziova**
Related to packit/packit#1004

### CODE REVIEW

File: content/docs/configuration.md [hunk 1]
```diff
@@ -166,6 +166,11 @@ notifications:
 
 You can disable the commenting by setting `successful_build` to `false`.
 
+#### copy_upstream_release_description
+(*bool*) When doing a new update in Fedora dist-git, the upstream release description
+is copied to the spec-file changelog when set to `true`. By default (`false`), 
+all messages of new commits are copied.
```
Comment (TomasTomecek) [inline content/docs/configuration.md:172]: I'm wondering if it makes sense to provide an example here.

File: content/docs/configuration.md [hunk 2]
```diff
@@ -166,6 +166,11 @@ notifications:
 
 You can disable the commenting by setting `successful_build` to `false`.
 
+#### copy_upstream_release_description
+(*bool*) When doing a new update in Fedora dist-git, the upstream release description
```
Comment (TomasTomecek) [inline content/docs/configuration.md:170]:
```suggestion
(*bool*) When doing a new update in Fedora dist-git, the GitHub upstream release description
```

File: content/docs/configuration.md (hunk 1 above)
Comment (lbarcziova) [reply content/docs/configuration.md:172]: I updated it, WDYT?

**Review by TomasTomecek — APPROVED** [review]
I like it, very nice!

**Pull request merged**